test/common/router/router_upstream_log_test.cc

```diff
@@ -91,6 +91,7 @@ class RouterUpstreamLogTest : public testing::Test {
                                    router_proto));
     router_.reset(new TestFilter(*config_));
     router_->setDecoderFilterCallbacks(callbacks_);
+    EXPECT_CALL(callbacks_.dispatcher_, setTrackedObject(_)).Times(testing::AnyNumber());
```

Comment:
Sorry can you add a similar comment here about the strict mock? Fine to do in your next change if you want.